src/SFA.DAS.Commitments.Support.SubSite/DependencyResolution/LoggingPropertyFactory.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+
+namespace SFA.DAS.Commitments.Support.SubSite.DependencyResolution
+{
+    [ExcludeFromCodeCoverage]
+    public class LoggingPropertyFactory : ILoggingPropertyFactory
+    {
+        public IDictionary<string, object> GetProperties()
+        {
+            var properties = new Dictionary<string, object>();
+            try
+            {
+                properties.Add("Version", GetVersion());
+            }
+            catch (Exception e)
```

Comment:
This line is emitting a warning, but the whole try..catch block here is redundant anyway and can be removed.